agent/dehydrate.js

```diff
@@ -165,4 +173,11 @@ function dehydrate(data: Object, cleaned: Array<Array<string>>, path?: Array<str
   }
 }
 
+function isPropertyGetter(obj, prop) {
+  /* eslint-disable no-proto */
+  const descriptor = Object.getOwnPropertyDescriptor(obj.hasOwnProperty(prop) ? obj : obj.__proto__, prop);
```

Comment:
Why is this check one level deep? Could it be on `__proto__.__proto__`, `__proto__.__proto__.__proto__`, etc? Do we need a loop here?